python/mxnet/_numpy_op_doc.py

```diff
@@ -699,6 +699,71 @@ def _npx_index_add(a, ind, val):
     pass
 
 
+def _npx_index_update(a, ind, val):
+    """
+    Update values to input according to given indexes.
+    If multiple indices refer to the same location it is undefined which update is chosen; it may choose
+    the order of updates arbitrarily and nondeterministically (e.g., due to concurrent updates on some
+    hardware platforms). Recommend not to use repeate positions.
```

Comment:
You may add a blank line here. (I find that `index_add` misses the blank line also).